first-read-last-write-cache/src/cache.rs

```diff
@@ -3,6 +3,7 @@ use crate::{CacheKey, CacheValue};
 use std::collections::{hash_map::Entry, HashMap};
 use std::sync::Arc;
 use thiserror::Error;
+use proptest::prelude::*;
```

Comment:
This import should go to `mod tests`.

Reply:
Thanks, I can't see this change in `Files changed` on github, have you pushed the commit?